src/main/kotlin/racing/view/ResultView.kt

```diff
@@ -0,0 +1,45 @@
+package racing.view
+
+import racing.car.Car
+import racing.common.CarRaceProperty
+
+class ResultView(private val carRaceProperty: CarRaceProperty) {
+
+    fun printResultByRound(round: Int) {
+        println("-- [$round]회차 결과 ------")
+        carRaceProperty.cars.map { printResultByCar(it) }
+        println("--------------------")
+    }
+
+    private fun printResultByCar(car: Car) {
+        println("${car.getCarName()} : ${getForwardMoveMark(car.getMoveCount())}")
+    }
+
+    private fun getForwardMoveMark(move: Int): String {
+        var forwardMark = ""
+        repeat(move) {
+            forwardMark += MARK_FORWARD_MOVE
+        }
+        return forwardMark
+    }
```

Comment:
```suggestion
    private fun getForwardMoveMark(move: Int): String {
        return MARK_FORWARD_MOVE.repeat(move)
    }
```
다음과 같이 작성하면 불필요한 변수 선언을 줄일 수 있습니다.

Reply:
오 좋은 팁입니다!
감사합니다 ㅎㅎ